logger.go

```diff
@@ -211,6 +211,20 @@ func (logger *Logger) Panicln(args ...interface{}) {
 	}
 }
 
+func (logger *Logger) GetHooks() LevelHooks {
+	var hooks LevelHooks
```

Comment:
Is there any reason for an extra variable, and not simply:
```go
func (logger *Logger) GetHooks() LevelHooks {
  logger.mu.RLock()
  defer logger.mu.RUnlock()
  return logger.Hooks
}
```
?

Reply:
defer is slightly slower on super frequent operations